docs/codeql/codeql-cli/about-ql-packs.rst

```diff
@@ -7,10 +7,8 @@ QL packs are used to organize the files used in CodeQL analysis. They
 contain queries, library files, query suites, and important metadata.
 
 The `CodeQL repository <https://github.com/github/codeql>`__ contains QL packs for
```

Comment:
Unfortunately, it wouldn't let me make a suggestion across lines 9 and 10. So this is half a suggestion.

🤔 We might want to simplify this so that we don't have to update it every time we add a language.

```suggestion
The `CodeQL repository <https://github.com/github/codeql>`__ contains standard QL packs for all supported languages.
```